tools/build/docker/install-everything.sh

```diff
@@ -4,16 +4,26 @@
 apk update
 apk add perl perl-io-socket-ssl perl-dev redis libarchive-dev libbz2 openssl-dev zlib-dev
 apk add imagemagick imagemagick-perlmagick libwebp-tools libheif
-apk add g++ make pkgconf gnupg wget curl nodejs nodejs-npm
+apk add g++ make pkgconf gnupg wget curl nodejs nodejs-npm file
 apk add shadow s6 s6-portable-utils
 
 #Hey it's cpanm
 curl -L https://cpanmin.us | perl - App::cpanminus
 
+#!AVX
+if [ $(uname -m) == 'x86_64' ]; then
+  cpanm --notest --installdeps Alien::FFI
```

Comment:
IIRC this just installs the required dependency packages and not the package itself, right? 

This whole block could use a bit more comments since it's kinda jank and might be forgotten otherwise. 😅

Reply:
Yes, only deps.

Sure 😅 I'll add comments.